sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ChangeFeedQueryImpl.java

```diff
@@ -95,7 +96,7 @@ private RxDocumentServiceRequest createDocumentServiceRequest(String continuatio
         }
 
         if(options.getStartDateTime() != null){
-            String dateTimeInHttpFormat = Utils.zonedDateTimeAsUTCRFC1123(options.getStartDateTime());
+            String dateTimeInHttpFormat = Utils.zonedDateTimeAsUTCRFC1123(options.getStartDateTime().atOffset(ZoneOffset.UTC));
```

Comment:
Instead of converting instant to OffsetDateTime, its better to add this helper method in Utils.java which directly takes Instant and formats it. 

```
public static String instantAsUTCRFC1123(Instant instant){
       return Utils.RFC_1123_DATE_TIME.format(instant);
}
```

Reply:
DateTime format cannot parse with Instant , we need zone time info , it will give runtime exception. Also we it cant be compatible with existing lease if we change the format.
https://stackoverflow.com/questions/25229124/format-instant-to-string